packages/npm-package-json-lint-config/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Master
+
+### New Feature
+
+- Added `type` and `react-native` to the order of preferred properties. 
```

Comment:
Might this be considered a "Breaking Change", if suddenly a project upgrades and they receive errors?

A sort of tricky matter in these lint tools, since technically most any change can be considered breaking.

Reply:
Well, I noticed that ESLint adds new default rules as new features. So I followed the same approach. It's usually easy to fix as you will see all errors/warnings listed. However, in the context of config, I think you are correct. We should consider it as a braking change.

Reply:
Addressed in https://github.com/WordPress/gutenberg/pull/15879/commits/8b48a1376c2dc88f88e0c5e9091597fc18d0b9e0.

Reply:
> Well, I noticed that ESLint adds new default rules as new features.

I think it depends whether these are expected to automatically take effect. For example, I would expect if they changed their `recommended` set, it would be treated as breaking. But if it's just a new rule, then "New Feature" is fine enough.

For context, I've been treating changes to the recommended set as breaking in `@wordpress/eslint-plugin`:

https://github.com/WordPress/gutenberg/blob/master/packages/eslint-plugin/CHANGELOG.md#breaking-changes